.drone/drone.yml

```diff
@@ -93,14 +93,14 @@ steps:
   depends_on:
   - clone
   environment: {}
-  image: grafana/loki-build-image:0.28.1
+  image: grafana/loki-build-image:0.28.2
```

Comment:
This is stale; it should be `0.28.3`

Reply:
That'll be updated on a separate PR where I'll cherry pick https://github.com/grafana/loki/pull/9372

Reply:
Done in this PR